### ISSUE TITLE
Again: Using wildcards in Wordpress ingress configuration: YAML parse error on wordpress/templates/ingress.yaml

### ISSUE THREAD

**Issue by floek** [bitnami/charts#5079]
**Which chart**:
wordpress 10.4.4

**Describe the bug**
When I add a wildcard entry like "*.domain.tld" to ingress.extraHosts I get the following error:
```console
Error: UPGRADE FAILED: YAML parse error on wordpress/templates/ingress.yaml: error converting YAML to JSON: yaml: line 259: did not find expected alphabetic or numeric character
```

This was already fixed with #4839 and maybe came back with #4953.

**To Reproduce**

1. add a domain with a wildcard to ingress.extraHosts

**Expected behavior**
I want to configure a subdomain multisite installation and need to use wildcard subdomains with cert_manager DNS validation.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.3.4", GitCommit:"a61ce5633af99708171414353ed49547cf05013d", GitTreeState:"clean", GoVersion:"go1.14.9"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7", GitCommit:"1dd5338295409edcfff11505e7bb246f0d325d15", GitTreeState:"clean", BuildDate:"2021-01-13T13:23:52Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7", GitCommit:"1dd5338295409edcfff11505e7bb246f0d325d15", GitTreeState:"clean", BuildDate:"2021-01-13T13:15:20Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
-

**Comment by alemorcuq** [comment]
Sorry about that. I will fix it ASAP.

**Comment by alemorcuq** [comment]
Hi, @floek. The PR has been merged, so this should be fixed when version `10.4.5` is published. I've also added a test to ensure this does not happen again.
